python_ta/reporters/node_printers.py

```diff
@@ -614,6 +614,36 @@ def render_pep8_errors_e306(msg, _node, source_lines=None):
     yield from render_context(msg.line, msg.line + 2, source_lines)
 
 
+def render_missing_return_statements(msg, _node, source_lines=None):
+    """
+    Render a missing return statements message
+    """
+    line = msg.line
+    end = _node.tolineno
+
+    # render function header and context
```

Comment:
While this was not a bad suggestion from the group, it's quite different in style than other PythonTA error rendering. Instead of doing this, I recommend passing the *function name* in the `args` to the message, and then displaying it in the text of the message. (As a comment, this might actually be good to do for other checks as well, but that's another matter.)